ontology/yaml/resources/ELECTRICAL/entity_types/UPS.yaml

```diff
@@ -66,6 +66,15 @@ UPS_IOBM_PPWM_UPSBM:
   - IOBM
   - PPWM
   - UPSBM
+  
+UPS_SS_BBM:
+  guid: "adcf545f-eb61-4130-a0f1-d611ab303d17"
+  description: "UPS unit with load bypass and battery mode and alarm."
+  is_canonical: true
+  implements:
+  - UPS
+  - SS
+  - BBM
```

Comment:
The functionality you are trying to model with `BBM` is already captured with the existing `UPSBM` type. Please use this type instead and make missing fields in your model if there are instances where the type requires a field that you do not actually have.

Reply:
noted ma'am, will remove this. usually it is only alarms that get affected.